mastodon/src/main/java/org/joinmastodon/android/fragments/SettingsFragment.java

```diff
@@ -296,6 +296,10 @@ public void onCreate(Bundle savedInstanceState){
 			GlobalUserPreferences.showNoAltIndicator=i.checked;
 			GlobalUserPreferences.save();
 		}));
+		items.add(new SwitchItem(R.string.sk_settings_show_posts_without_alt, R.drawable.ic_fluent_eye_tracking_on_24_regular, GlobalUserPreferences.showPostsWithoutAlt, i->{
```

Comment:
I think this setting should include a summary, explaining to the user what it does. What do you think @LucasGGamerM?

Reply:
summary added